### DISCUSSION TITLE
I can't download release from nextcloud/richdocuments

### DISCUSSION THREAD

**Discussion by djooberlee** [community/community#59681] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Feature Area

Issues

### Body

https://github.com/nextcloud/richdocuments/releases/download/v6.3.6/richdocuments.tar.gz

![image](https://github.com/community/community/assets/18099852/654e5bbc-584b-4d14-ba62-7511d6502e50)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by chris97pl** [comment]
I can confirm that various Github systems are down at the moment. I have the whole build process failing because of that.

**Comment by hrQAQ** [comment]
I have the same problem downloading release from https://github.com/OI-wiki/OI-Wiki-export/releases/tag/v20230625_0923, hope the service fix quickly

**Comment by djooberlee** [comment]
resolved
